src/components/about/Card.jsx

```diff
@@ -1,7 +1,21 @@
 import React from "react";
+import { FaDiscord } from "react-icons/fa";
 
 const Card = () => {
```

Comment:
CSS is good. Now add props: {text, icon, link} 

example of how to use these props: https://github.com/acm-ucr/asme-website/blob/dev/src/components/home/JoinCard.jsx